### PR TITLE
E2e/feature/ri 3638 big keys

### PR DESCRIPTION
**Pull request by AlenaNaboka**
1. Added dot at the end of 2 sentences:
image
2. Created e2e for top keys feature

### CODE REVIEW

**Review by ViktarStarastsenka — APPROVED** [review]

**Review by vlad-dargel — APPROVED** [review]

File: tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    const newPromoButtonText = chance.word({ length: 10 });
+
+    //Edit json file values
+    contentPromoButtonFile.set('cloud.title', newPromoButtonText);
+    contentPromoButtonFile.set('cloud.description', newPromoButtonText);
+    contentPromoButtonFile.save();
+    timestampPromoButtonFile.set('timestamp', newTimestamp);
+    timestampPromoButtonFile.save();
+
+    fixture `Automatically update information`
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts:32]: the same fixture name as in previous fixture above

File: tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            // Check the promo button after the opening of app
+            await t.expect(myRedisDatabasePage.promoButton.textContent).notContains(newPromoButtonText, 'Promo button text is updated');
+            // Get the values from build.json and create-redis.json files
+            const actualTimestamp = await timestampPathNew.get('timestamp');
+            const actualPromoButtonTitle = await contentPathNew.get('cloud.title');
+            const actualPromoButtonDescription = await contentPathNew.get('cloud.description');
+            // Check the json files are automatically updated
+            console.log(`timestamp: ${actualTimestamp}`);
+            console.log(`actualPromoButtonTitle: ${actualPromoButtonTitle}`);
+            console.log(`actualPromoButtonDescription: ${actualPromoButtonDescription}`);
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts:52]: console.log

File: tests/e2e/tests/critical-path/memory-efficiency/top-keys-table.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+test
+    .before(async t => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        // Create keys
+        await populateDBWithHashes(ossStandaloneConfig.host, ossStandaloneConfig.port, keyToAddParameters);
+        // Go to Analysis Tools page
+        await t.click(myRedisDatabasePage.analysisPageButton);
+    })
+    .after(async() => {
+        await cliPage.sendCommandInCli('flushdb');
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/memory-efficiency/top-keys-table.e2e.ts:37]: I think that it's not good to delete all keys in this db if in future we will make testcafe parallelizm
also we have this function to delete all keys from db:
`await deleteAllKeysFromDB(dbParameters.host, dbParameters.port);`

so it's better to use ossStandaloneRedisearch db here for me

File: tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts (hunk 2 above)
Comment (AlenaNaboka) [reply tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts:52]: removed

File: tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts (hunk 1 above)
Comment (AlenaNaboka) [reply tests/e2e/tests/critical-path/files-auto-update/promo-button-autoupdate.e2e.ts:32]: changed, thanks

File: tests/e2e/tests/critical-path/memory-efficiency/top-keys-table.e2e.ts (hunk 3 above)
Comment (AlenaNaboka) [reply tests/e2e/tests/critical-path/memory-efficiency/top-keys-table.e2e.ts:37]: We have 4 different threads and for each thread we create new RTE and tests are not interacted between each other so I think that flashing all db in one command will not increase tests run. Vise verse I would delete via UI ~20 keys.
Function "deleteAllKeysFromDB" is also based on flushdb command

Comment (AlenaNaboka) [reply tests/e2e/tests/critical-path/memory-efficiency/top-keys-table.e2e.ts:37]: fixed

**Review by vlad-dargel — APPROVED** [review]

**Pull request merged**